libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorAdaptiveRecognizer.cs

```diff
@@ -226,12 +226,16 @@ private void InitializeModel()
         {
             if (_modelFolder == null)
             {
-                throw new ArgumentNullException($"Missing `ModelFolder` information.");
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
```

Comment:
I know this is a cherry pick but I think it may have been better to use InvalidOperationException here with a descriptive message rather than argumentnullexception with a pragma ([The exception that is thrown when a method call is invalid for the object's current state.](https://docs.microsoft.com/en-us/dotnet/api/system.invalidoperationexception?view=net-5.0)).